### DISCUSSION TITLE
How to run docker image in GHA which was built in previous step (without push/pull interaction with a registry)?

### DISCUSSION THREAD

**Discussion by geoHeil** [community/community#30767] [Actions]
Can you help me answer:  https://stackoverflow.com/questions/73484162/docker-run-image-from-gha-or-local-cache

- I have a step which builds the image
- and a second step which executes a command in the image. However, this second step is only working fine when it can pull the image from a remote registry. Why can`t I access the image from the previous step?

**Comment by ViacheslavKudinov** [comment]
Hi,
I saw you got the answer on stackoverflow.
There is also some snippet for that https://github.com/docker/build-push-action/blob/master/docs/advanced/export-docker.md

**Reply by sayhiben** [reply]
Specifically, `load: true`, which is short for `--output=type=docker` described [here](https://github.com/docker/buildx/blob/master/docs/reference/buildx_build.md#docker):

> The docker export type writes the single-platform result image as a [Docker image specification](https://github.com/docker/docker/blob/v20.10.2/image/spec/v1.2.md) tarball on the client. Tarballs created by this exporter are also OCI compatible.
